src/extensionsConfig.js

```diff
@@ -0,0 +1,71 @@
+import _keys from 'lodash/keys';
+
+const fields = {
+  'accelerator_experiments:accelerator': {
+    type: 'string',
+    default: '',
+  },
+  'accelerator_experiments:curated_relation': {
+    type: 'boolean',
+    default: false,
+  },
+  'accelerator_experiments:experiment': {
+    type: 'string',
+    default: '',
+  },
+  'accelerator_experiments:institution': {
+    type: 'string',
+    default: '',
+  },
+  'accelerator_experiments:legacy_name': {
+    type: 'string',
+    default: '',
+  },
+  'accelerator_experiments:project': {
+    type: 'string',
+    default: '',
+  },
+  'accelerator_experiments:study': {
+    type: 'string',
+    default: '',
+  },
+  'standard_CERN_status:CERN_applicability': {
+    type: 'string',
+    default: '',
+    isVisible: false,
```

Comment:
why the  `isVisible`?

Reply:
`isVisible` controls if the field will be displayed in the frontsite to the Patron. If we do not configure the field, then we cannot show it in the backoffice form. I am more than open to ideas about controlling that behaviour.